doc/source/docker/run-as-subprocess.rst

```diff
@@ -1,53 +1,113 @@
+   not attempt to create or manage these processes. Instead, they must be started
+   externally.
 
-       FROM flwr/supernode:|stable_flwr_version|
+Both modes can be mixed for added flexibility. For instance, you can run the SuperLink
+in ``subprocess`` mode while keeping the SuperNode in ``process`` mode, or vice versa.
 
-       WORKDIR /app
-       COPY pyproject.toml .
-       RUN sed -i 's/.*flwr\[simulation\].*//' pyproject.toml \
-           && python -m pip install -U --no-cache-dir .
+To run the SuperLink and SuperNode in isolation mode ``process``, refer to the
+:doc:`tutorial-quickstart-docker` guide. To run them in ``subprocess`` mode, follow the
+instructions below.
 
-       ENTRYPOINT ["flower-supernode"]
+.. tab-set::
 
-2. Next, build the SuperNode Docker image by running the following command in the
-   directory where Dockerfile is located:
+    .. tab-item:: ClientApp
 
-   .. code-block:: shell
+        **Prerequisites**
 
-       $ docker build -f Dockerfile.supernode -t flwr_supernode:0.0.1 .
+        1. Before running the ClientApp as a subprocess, ensure that the FAB dependencies have
+        been installed in the SuperNode images. This can be done by extending the SuperNode
+        image:
 
-Run the ClientApp as a Subprocess
----------------------------------
+        .. code-block:: dockerfile
+            :caption: supernode.Dockerfile
+            :linenos:
+            :substitutions:
 
-Start the SuperNode with the flag ``--isolation subprocess``, which tells the SuperNode
-to execute the ClientApp as a subprocess:
+            FROM flwr/supernode:|stable_flwr_version|
 
-.. code-block:: shell
+            WORKDIR /app
+            COPY pyproject.toml .
+            RUN sed -i 's/.*flwr\[simulation\].*//' pyproject.toml \
+                && python -m pip install -U --no-cache-dir .
 
-    $ docker run --rm \
-        --detach \
-        flwr_supernode:0.0.1 \
-        --insecure \
-        --superlink superlink:9092 \
-        --node-config "partition-id=1 num-partitions=2" \
-        --supernode-address localhost:9094 \
-        --isolation subprocess
+            ENTRYPOINT ["flower-supernode"]
+
+        2. Next, build the SuperNode Docker image by running the following command in the
+        directory where Dockerfile is located:
+
+        .. code-block:: shell
+
+            $ docker build -f supernode.Dockerfile -t flwr_supernode:0.0.1 .
+
+        **Run the ClientApp as a Subprocess**
+
+        Start the SuperNode and run the ClientApp as a subprocess (note that
+        the subprocess mode is the default, so you do not have to explicitly set the ``--isolation`` flag):
+
+        .. code-block:: shell
+
+            $ docker run --rm \
+                --detach \
+                flwr_supernode:0.0.1 \
+                --insecure \
+                --superlink <superlink-address>:9092
+
+    .. tab-item:: ServerApp
```

Comment:
Let's swap the order and have `ServerApp` at the top, followed by `ClientApp`. This makes it consistent with the title.